src/shapes/object.class.js

```diff
@@ -748,6 +748,9 @@
      * @param {Boolean} fromLeft When true, context is transformed to object's top/left corner. This is used when rendering text on Node
      */
     transform: function(ctx, fromLeft) {
+      if (this.group && this.canvas.preserveObjectStacking && this.group === this.canvas._activeGroup) {
```

Comment:
This is breaking for me right now because this.cavas doesn't seem to be defined, which is odd because all groups should have a canvas.